src/routes/categories/containers/__tests__/__snapshots__/CategoriesPage.spec.js.snap

```diff
@@ -90,7 +90,7 @@ exports[`CategoriesPage should match snapshot and render a Page if page has no c
   <Page
     content="exampleContent0"
     language="en"
-    lastUpdate={"2016-01-07T09:36:24.000Z"}
+    lastUpdate={"2016-01-07T08:36:24.000Z"}
```

Comment:
We may have to set a timezone for jest: https://stackoverflow.com/questions/56261381/how-do-i-set-a-timezone-in-my-jest-config

Reply:
Thanks. should be Fixed. I used the config below to change the tz and pushed the updated snapshots without the code changes to the jest configs as I'm not sure it should be in this pr or not
```js
module.exports = async () => {
  process.env.TZ = 'Europe/Berlin'
}
```

Reply:
@maxammann what do you think? Should this be pushed? I'm inclined to say yes because we had this problem already some times.

Reply:
@maxammann @klinzo Please let me know when you decide on this